DcCore/DcCore/DC/events.swift

```diff
@@ -56,118 +56,105 @@ public class DcEventHandler {
 
         case DC_EVENT_CONFIGURE_PROGRESS:
             logger.info("📡[\(accountId)] configure: \(Int(data1))")
-            DispatchQueue.main.async {
-                let done = Int(data1) == 1000
-
-                NotificationCenter.default.post(name: Event.configurationProgress, object: nil, userInfo: [
-                    "progress": Int(data1),
-                    "error": Int(data1) == 0,
-                    "done": done,
-                    "errorMessage": event.data2String,
-                ])
-
-                if done {
-                    UserDefaults.standard.set(true, forKey: Constants.Keys.deltachatUserProvidedCredentialsKey)
-                    UserDefaults.standard.synchronize()
-                }
-            }
-
+            
+            let done = Int(data1) == 1000
+            
+            NotificationCenter.default.post(name: Event.configurationProgress, object: nil, userInfo: [
+                "progress": Int(data1),
+                "error": Int(data1) == 0,
+                "done": done,
+                "errorMessage": event.data2String,
+            ])
+            
+            if done {
+                UserDefaults.standard.set(true, forKey: Constants.Keys.deltachatUserProvidedCredentialsKey)
+            }
+            
         case DC_EVENT_IMEX_PROGRESS:
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: Event.importExportProgress, object: nil, userInfo: [
-                    "progress": Int(data1),
-                    "error": Int(data1) == 0,
-                    "done": Int(data1) == 1000,
-                    "errorMessage": self.dcAccounts.get(id: accountId).lastErrorString,
-                ])
-            }
+            
+            NotificationCenter.default.post(name: Event.importExportProgress, object: nil, userInfo: [
+                "progress": Int(data1),
+                "error": Int(data1) == 0,
+                "done": Int(data1) == 1000,
+                "errorMessage": self.dcAccounts.get(id: accountId).lastErrorString,
+            ])
         case DC_EVENT_MSGS_CHANGED:
```

Comment:
```suggestion
            ])

        case DC_EVENT_MSGS_CHANGED:
```

(just normalising whitespace)